packages/manager/apps/pci-cold-archive/src/pages/containers/add-user/AddUser.page.tsx

```diff
@@ -19,13 +19,21 @@ import { TUser } from '@/api/data/users';
 import StepOneComponent from './StepOne.component';
 import StepTwoComponent from './StepTwo.component';
 import { useProductRegionsAvailability } from '@/api/hooks/useRegions';
+import {
+  OBJECT_CONTAINER_USER_ROLE_ADMIN,
+  OBJECT_CONTAINER_USER_ROLES,
+} from '@/constants';
+
+type TFoo = typeof OBJECT_CONTAINER_USER_ROLES[number];
```

Comment:
I wrote `TFoo` as an example, but here I don't think it's suitable 😅

Reply:
Any news on that ?

Reply:
it's fixed